spec/mainadditions.html

```diff
@@ -271,6 +271,266 @@ <h1>AvailableNamedTimeZoneIdentifiers ( ): a List of Time Zone Identifier Record
+      </emu-note>
+    </emu-clause>
+
+    <p>[...]</p>
+
+    <emu-clause id="sec-timezoneestring" type="abstract operation">
+      <h1>
+        TimeZoneString (
+          _tv_: an integral Number,
+        ): a String
+      </h1>
+      <dl class="header">
+      </dl>
+      <emu-alg>
+        1. Let _systemTimeZoneIdentifier_ be SystemTimeZoneIdentifier().
+        1. <ins>Let _parseResult_ be ! ParseTimeZoneIdentifier(_systemTimeZoneIdentifier_).</ins>
+        1. If <del>IsTimeZoneOffsetString(_systemTimeZoneIdentifier_) is *true*</del><ins>_parseResult_.[[OffsetNanoseconds]] is not ~empty~</ins>, then
+          1. Let _offsetNs_ be <del>ParseTimeZoneOffsetString(_systemTimeZoneIdentifier_)</del><ins>_parseResult_.[[OffsetNanoseconds]]</ins>.
+        1. Else,
+          1. Let _offsetNs_ be GetNamedTimeZoneOffsetNanoseconds(_systemTimeZoneIdentifier_, ℤ(ℝ(_tv_) × 10<sup>6</sup>)).
+        1. <del>Let _offset_ be 𝔽(truncate(_offsetNs_ / 10<sup>6</sup>)).</del>
+        1. <del>If _offset_ is *+0*<sub>𝔽</sub> or _offset_ > *+0*<sub>𝔽</sub>, then</del>
+          1. <del>Let _offsetSign_ be *"+"*.</del>
+          1. <del>Let _absOffset_ be _offset_.</del>
+        1. <del>Else,</del>
+          1. <del>Let _offsetSign_ be *"-"*.</del>
+          1. <del>Let _absOffset_ be -_offset_.</del>
+        1. <del>Let _offsetMin_ be ToZeroPaddedDecimalString(ℝ(MinFromTime(_absOffset_)), 2).</del>
+        1. <del>Let _offsetHour_ be ToZeroPaddedDecimalString(ℝ(HourFromTime(_absOffset_)), 2).</del>
+        1. <ins>Let _offsetString_ be FormatOffsetTimeZoneIdentifier(_offsetNs_, ~legacy~).</ins>
+        1. Let _tzName_ be an implementation-defined string that is either the empty String or the string-concatenation of the code unit 0x0020 (SPACE), the code unit 0x0028 (LEFT PARENTHESIS), an implementation-defined timezone name, and the code unit 0x0029 (RIGHT PARENTHESIS).
+        1. <del>Return the string-concatenation of _offsetSign_, _offsetHour_, _offsetMin_, and _tzName_.</del>
+        1. <ins>Return the string-concatenation of _offsetString_ and _tzName_.</ins>
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-isoffsettimezoneidentifier" oldids="sec-istimezoneoffsetstring" type="abstract operation">
+      <h1>
+        IsOffsetTimeZoneIdentifier (
+          _offsetString_: a String,
+        ): a Boolean
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>The return value indicates whether _offsetString_ conforms to the grammar given by <del>|UTCOffset|</del><ins>|TimeZoneUTCOffsetName|</ins>.</dd>
+      </dl>
+      <emu-note type="editor">
+        This abstract operation was renamed from IsTimeZoneOffsetString.
+        After https://github.com/tc39/ecmarkup/issues/530 is fixed, this note can be replaced with `ins` and `del` in the AO name above.
+      </emu-note>
```

Comment:
Yeah, I tried that earlier and it didn't work.  I think I tried every possible permutation and nothing worked. Here's what I get from using the spec text pattern in your comment for both AOs:

<img width="988" alt="image" src="https://github.com/tc39/proposal-temporal/assets/277214/0ef7a269-3913-4b67-9303-b2d728d8d7ec">

I'm not sure why the first AO shows the `<ins>` text in green while the second one doesn't. But the return types are missing in both, so it's hosed regardless.

I'd like to get this PR merged tonight if possible, and I think I can resolve all the other issues tonight. So I'll plan to leave the notes as-is and we can revise it once ecmarkup is fixed.

Reply:
Lol, I realized that the lack of the return type is fine, because it's shown in prose in the new name but shown in the H1 in the `<del>` block.  

The missing green in the second AO was caused by me forgetting `class="blockl"`.

So I'll remove the notes and just use ins/del. 

I'll follow up over in ecmarkup with some minor complaints about the implementation.

Reply:
> I'll follow up over in ecmarkup with some minor complaints about the implementation.

https://github.com/tc39/ecmarkup/issues/537